CHANGELOG.md

```diff
@@ -21,6 +21,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add group authorization to PUT/events/{event_id}/images/{image_id} endpoint.[#719](https://github.com/rokwire/rokwire-building-blocks-api/issues/719)
 - Add group authorization to DELETE /events/{event_id}/images/{image_id} endpoint.[#720](https://github.com/rokwire/rokwire-building-blocks-api/issues/720)
 - Make catalog pages to show different items for reviewers and non reviewers. [#709](https://github.com/rokwire/rokwire-building-blocks-api/issues/709)
+- Make catalog pages to show all the items for contribution admins. [#721](https://github.com/rokwire/rokwire-building-blocks-api/issues/721)
+- Make contribution details pages to show all items for reviewers and admins. [#728](https://github.com/rokwire/rokwire-building-blocks-api/issues/728)
```

Comment:
Please move all of the new CHANGELOG entries need to the Unreleased section. Thanks.

Reply:
Moved to unreleased... sorry

Reply:
No issues. Thank you.